.github/workflows/lint-test.yml

```diff
@@ -11,16 +11,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [16.x]
+-    strategy:
+-      matrix:
+-        node-version: [16, 18]
```

Comment:
since this is tested on these two major, consider having the package.json specify both like https://github.com/Agoric/agoric-sdk/pull/7293/ does

Reply:
I actually reduced down to a matrix of just 16 for now. I do see where having package.json at the root of the repo match well with the CI makes perfect sense.